src/modules/EventList/index.tsx

```diff
@@ -71,15 +74,190 @@ export interface EventListUIProps {
     scrollRef: RefObject<HTMLDivElement>;
     searchTerm?: string;
     setFocus?: Dispatch<SetStateAction<number>>;
+    useConfigOptions: boolean;
 }
 
+interface EventRowProps {
+    refetch?: () => void;
+    event: EventListEvent;
+    maxHits?: number;
+    numMentions: number | null | undefined;
+    index: number;
+    isRefetching: boolean;
+    onSelectEvent?: ChangeHandler<EventListEvent>;
+    renderedRefetch: boolean;
+    searchTerm?: string;
+    setFocus?: Dispatch<SetStateAction<number>>;
+    showDivider: boolean;
+}
+
+const EventRow = ({
+    event,
+    numMentions,
+    maxHits = 0,
+    isRefetching,
+    onSelectEvent,
+    refetch,
+    renderedRefetch,
+    searchTerm,
+    index,
+    setFocus,
+    showDivider,
+}: EventRowProps) => {
+    const hitRatio = (numMentions || 0) / maxHits;
```

Comment:
If `maxHits` is 0, you'll get a `NaN` here, so might be worth making this a ternary so when `maxHits` is 0, then `hitRatio` can be 0 too